specification/http/src/main/scripts/org/kaazing/specification/http/rfc7230/message.format/head.response.must.not.have.content.though.may.have.content.length/response.rpt

```diff
@@ -5,7 +5,7 @@ connected
 read method "HEAD"
 read closed
 
-write status "100" "OK"
+write status "200" "OK"
 write header "Content-Length" "100"
 write close
```

Comment:
Yes, sort of.  It was generally expected that when writing http level scripts that k3po could do anything valid at the tcp layer.  Closing the connection here is valid so its okay.  We have never implemented semantics to control the tcp layer, from a k3po script that is written in the http layer